BrushedMotorControllerFirmware/Kconfigs/Kconfig.speed_cntrl

```diff
@@ -1,11 +1,19 @@
 menu "Speed Control Configuration"
 
+config SPEED_CONTROL_ENABLE
```

Comment:
same as above, please use `if` statement for multiple configs that depend on the same config